agent/bls.mjs

```diff
@@ -374,6 +407,7 @@ function perform_bls_glue(
 }
 
 function perform_bls_glue_u256( details, u256, arrSignResults ) {
+    const imaState = state.get();
```

Comment:
Please, modify duplicated code of `perform_bls_glue_u256` and `perform_bls_glue`, according to https://github.com/skalenetwork/internal-support/blob/main/docs/wiki/code-guidelines.md#coherent-style

Reply:
Done